src/elisp/treemacs-impl.el

```diff
@@ -773,14 +778,20 @@ failed."
             (goto-char start))
         search-result))))
 
-(define-inline treemacs--goto-custom-top-level-node (path)
-  (inline-letevals (path)
-    (inline-quote
-     (-when-let (result (treemacs--find-custom-top-level-node ,path))
-       (treemacs--evade-image)
-       (hl-line-highlight)
-       (set-window-point (get-buffer-window) (point))
-       result))))
+(defun treemacs--find-custom-project-node (path)
+  "Move to the project extension node at PATH."
+  (unless (treemacs-project->is-expanded? (car path))
+    (treemacs--expand-root-node (car path)))
+  (or (-some-> (treemacs-find-in-dom path)
+               (treemacs-dom-node->position))
+      (let* ((project (car path))
+             (start (point))
+             (project-start (treemacs-project->position project))
```

Comment:
If you need a project's start and end positions you can use `treemacs--get-bounds-of-project`.

Reply:
Thanks, I'll keep that in mind - I fixed it in an entirely different way, so that's no longer necessary here.